README.md

```diff
@@ -165,8 +165,8 @@ pip install pandas
 
 ## Dependencies
 - [NumPy](https://www.numpy.org): 1.13.3 or higher
-- [python-dateutil](https://labix.org/python-dateutil): 2.5.0 or higher
-- [pytz](https://pythonhosted.org/pytz): 2015.4 or higher
+- [python-dateutil](https://labix.org/python-dateutil): 2.6.1 or higher
```

Comment:
Instead of having to manage this in 2 places, I think we should just link to our docs.

Reply:
You mean, completely remove the "Dependencies" section from README.md, and put something like (sorry in advanced for my English):


### To see the minimum required versions:
click [here](https://pandas.pydata.org/pandas-docs/stable/install.html#dependencies)


______

Or there is an actual option to link the version from the web directly to the README.md?

Reply:
Maybe just remove the version numbers here and clarify the sentence below to say:

`See the full installation instructions for minimum supported versions of required, recommended and optional dependencies.`

Reply:
Ty so much @mroeschke , you saved me a huge headache of how to format the message.

Really appreciate it.